lib/styles/settings/_shadows.scss

```diff
@@ -1,19 +1,20 @@
 @import 'colors';
+@import 'colors/tokens';
 
-$shadow-s: 0 1px 2px $color-neutrals-shadow-weak, 0 8px 16px $color-neutrals-shadow-weak,
-	0 2px 4px $color-neutrals-shadow-strong, 0 0 4px $color-neutrals-shadow;
-$shadow-m: 0 2px 4px $color-neutrals-shadow-weak, 0 6px 12px $color-neutrals-shadow-strong,
-	0 0 4px $color-neutrals-shadow;
-$shadow-l: 0 4px 8px $color-neutrals-shadow-weak, 0 10px 20px $color-neutrals-shadow-strong,
-	0 0 4px $color-neutrals-shadow;
-$shadow-xl: 0 8px 16px $color-neutrals-shadow-weak, 0 16px 32px $color-neutrals-shadow-strong,
-	0 0 4px $color-neutrals-shadow;
+$shadow-s: 0 1px 2px $color-default-shadow-weak, 0 8px 16px $color-default-shadow-weak,
+	0 2px 4px $color-default-shadow-strong, 0 0 4px $color-default-shadow;
+$shadow-m: 0 2px 4px $color-default-shadow-weak, 0 6px 12px $color-default-shadow-strong,
+	0 0 4px $color-default-shadow;
+$shadow-l: 0 4px 8px $color-default-shadow-weak, 0 10px 20px $color-default-shadow-strong,
+	0 0 4px $color-default-shadow;
+$shadow-xl: 0 8px 16px $color-default-shadow-weak, 0 16px 32px $color-default-shadow-strong,
+	0 0 4px $color-default-shadow;
 
 // deprecated
-$default-shadow: 0 4px 12px rgba(0, 0, 0, 0.35);
-$default-lighter-shadow: 0 4px 12px rgba($color-firefly-black, 0.35);
-$small-shadow: 0 2px 4px rgba(0, 0, 0, 0.25);
-$medium-lightest-shadow: 0 4px 8px rgba($color-firefly-black, 0.12);
-$medium-lightest-shadow-up: 0 -4px 8px rgba($color-firefly-black, 0.12);
-$medium-lightest-shadow-top: inset 0 4px 8px rgba($color-firefly-black, 0.12);
-$medium-lightest-shadow-bottom: inset 0 -4px 8px rgba($color-firefly-black, 0.12);
+$default-shadow: 0 4px 12px rgba(var(--raw-black-rgb), 0.3);
+$default-lighter-shadow: 0 4px 12px rgba(var(--raw-black-rgb), 0.3);
```

Comment:
two variables with the same value? Can we migrate to one?

Reply:
Yes, I will remove second one.